language_utilities/constant.py

```diff
@@ -16,9 +16,17 @@
 DUTCH_LANG = 'nl'
 FRENCH_LANG = 'fr'
 GERMAN_LANG = 'de'
+ITALIAN_LANG = 'it'
+
+MALAY_LANG = 'ms'
+BAHASA_LANG = 'id'
+SWAHILI_LANG = 'sw'
+ARABIC_LANG = 'ar'
 
 # language translation status
 TRANSLATED_TEXT = 'translated_text'
 
-INDIC_LANGUAGES_SET = {HINDI_LANG, TAMIL_LANG, MARATHI_LANG, GUJARATI_LANG, KANNADA_LANG, TELEGU_LANG}
+INDIC_LANGUAGES_SET = {HINDI_LANG, TAMIL_LANG, MARATHI_LANG, GUJARATI_LANG, KANNADA_LANG, TELEGU_LANG, BENGALI_LANG,
+                       PUNJABI_LANG, MALAYALAM_LANG, ORIYA_LANG}
 EUROPEAN_LANGUAGES_SET = {SPANISH_LANG, DUTCH_LANG, FRENCH_LANG, GERMAN_LANG}
+OTHER_AVAILABLE_LANGUAGES = {MALAY_LANG, BAHASA_LANG, SWAHILI_LANG}
```

Comment:
These sets should be moved to name_detection.py, because they are used to make decisions specific for that module and can change